### PR TITLE
Create state interface Done

### PR DESCRIPTION
**Pull request by bw55555**
Done creating state interface

### CODE REVIEW

File: src/main/java/interfaces/State.java [hunk 1]
```diff
@@ -0,0 +1,8 @@
+package interfaces;
+
+public interface State {
```
Comment (mahirhassan-uoft) [inline src/main/java/interfaces/State.java:3]: cool

**Review by mahirhassan-uoft — APPROVED** [review]
nice

**Review by charlietaoxl — APPROVED** [review]
Looks good, has everything it needs.

**Review by marjaleenakiik — APPROVED** [review]
Test code approved, no issues

**Pull request merged**